docs/admin/kubeadm.md

```diff
@@ -127,8 +139,8 @@ for a full list of available versions).
 
 ### `kubeadm join`
 
-`kubeadm join` has one mandatory flag, the token used to secure cluster bootstrap,
-and one mandatory argument, the master IP address.
+`kubeadm join` has one mandatory flag, the token used to secure cluster
```

Comment:
The commas make this read like a list of four things instead of a list of two things. What's the difference between a flag and an argument?

Suggest a reword: "When you use `kubeadm join`, you must supply the token used to secure cluster boostrap as a mandatory flag, and the master IP address as a mandatory argument."